ext/AvizExt/theme.jl

```diff
@@ -11,7 +11,7 @@ const COLORS::Dict{Symbol,Symbol} = Dict(
 )
 
 
-function scenario_type(rs)
+function scenario_type(rs; scenarios=(:))
```

Comment:
@ConnectedSystems I don't get what does that `(:)` mean. Can you explain to me, please?

Reply:
colon operator to select all, e.g.,

```
some_dataframe[:, 1]
```